packages/dev/core/src/Gizmos/cameraGizmo.ts

```diff
@@ -160,6 +163,22 @@ export class CameraGizmo extends Gizmo implements ICameraGizmo {
     private static _Scale = 0.05;
     private _invProjection = new Matrix();
 
+    /**
+     * Disposes and replaces the current camera mesh in the gizmo with the specified mesh
+     * @param mesh The mesh to replace the default mesh of the camera gizmo
+     */
+    public setCustomMesh(mesh: Mesh) {

```

Comment:
maybe this function should be used in the `_CreateCameraMesh`, otherwise setting a custom mesh and then setting the camera will revert the custom mesh, but will not dispose it. 
In the same scenario - might be better to keep the custom mesh instead of replacing it again.

Reply:
huuum true true, in that case I'll use `_customMeshSet` to check if we should skip creating the camera mesh.